packages/styles/scss/components/card-group/_card-group.scss

```
@@ -162,6 +161,63 @@
     }
   }
 
+  :host(#{$dds-prefix}-card-group)[border='visible'] {
```

This would be changed to
```suggestion
  :host(#{$dds-prefix}-card-group)[border] {
```